### PR TITLE
pandoc 1.17.1

### PR DESCRIPTION
**Pull request by dunn**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

cc @ilovezfs

### CODE REVIEW

**Comment by ilovezfs** [comment]
:+1:

File: Formula/pandoc.rb [hunk 1]
```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```
Comment (rakkesh) [inline Formula/pandoc.rb]: Is this `dependency` really required here? Pandoc builds and works fine without `depends_on "gmp"`.

Comment (dunn) [inline Formula/pandoc.rb]: Looks like it's a runtime dep, according to https://github.com/jgm/pandoc/issues/2943#issuecomment-221423238

Comment (dunn) [inline Formula/pandoc.rb]: Also https://github.com/jgm/pandoc/issues/11#issuecomment-632751

Comment (rakkesh) [inline Formula/pandoc.rb]: Thanks for providing the reference. I haven't tried it on GNU/Linux but on Mac OS X I don't notice such an issue without installing `gmp`. 

```
$ pandoc -o test.html test.md      # works on all my machines
```

Also, `pandoc` doesn't link with `gmp` even with `depends_on "gmp"` in the formula.

```
$ otool -L `which pandoc`
/usr/local/bin/pandoc:
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.5)
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1226.10.1)
```

I have just now built the current version `1.17.1` without `depends_on "gmp"`. The generated binary works fine.

Here are the logs, https://gist.github.com/d0235638893c6074fa80e906589a0232

Comment (dunn) [inline Formula/pandoc.rb]: Alright, let's give it a shot.  If anything breaks we can revert.  Thanks for looking closer!

Comment (ilovezfs) [inline Formula/pandoc.rb]: I think this is a consequence of https://github.com/Homebrew/legacy-homebrew/pull/39134. I'm not sure why pandoc wasn't included in https://github.com/Homebrew/legacy-homebrew/pull/39709.

Comment (ilovezfs) [inline Formula/pandoc.rb]: @geoff-codes Perhaps you recall?

**Pull request closed without merging**